ledger/eval/eval.go

```diff
@@ -193,6 +206,56 @@ func (x *roundCowBase) lookup(addr basics.Address) (ledgercore.AccountData, erro
 	return ad, err
 }
 
+// balanceRound reproduces the way that the agreement package finds the round to
+// consider for online accounts.
+func (x *roundCowBase) balanceRound() (basics.Round, error) {
+	phdr, err := x.BlockHdr(agreement.ParamsRound(x.rnd))
```

Comment:
nit: idk, maybe `ParamsRound` and `BalanceRound` should now live some shared package like basics/protocol/config ?

Reply:
I'm willing to move it, but `agreement` seems reasonable to me as is, it seems like an `agreement` function.  Anyone have a specific preference?

Reply:
It does seems like these are inherently `agreement` functions. Is the objection that you want to avoid importing the agreement package here?